Alibaba OSS support in Multi-Cloud Gelato Project

Click2Cloud-Gamma opened the pull request:
In Gelato project of SODA foundation, currently it supports to perform object operations, lifecycle management and object migration on various cloud vendors like AWS, Azure, GCP, IBM, Huawei, YIG and Ceph. So adding Alibaba as a backend will allow user to perform object management, object lifecycle management (in-cloud & cross-cloud) and migration for Alibaba Cloud using Gelato(multi-cloud) Project for Alibaba OSS Service.

This is  design-spec for details of features that Alibaba OSS will support as new backend in Gelato Project

skdwriting commented:
@Click2Cloud-Gamma 
Why there is no description to the PR?
How about the previous PR #51 comments are closed? no comment updates

skdwriting commented on specs/daito/Alibaba_OSS_Support.md:
```diff
@@ -0,0 +1,329 @@
+
+| | Tier_1 | Tier_99 | Tier_999 |
+|---------------|---------------|---------------|---------------|
+| Alibaba OSS | STANDARD | IA | Archive |
```
please provide the details on the storage classes and attributes, compared to exsiting?

Click2Cloud-Gamma replied:
Added comparison with storage classes of other supported cloud. Thanks

skdwriting commented on specs/daito/Alibaba_OSS_Support.md:
```diff
@@ -0,0 +1,329 @@
+* Create OpenSDS Bucket on Alibaba OSS
+* Delete Bucket from Alibaba OSS
+* SSE supported
+* Upload Object on Alibaba OSS (PUT Object)  
```
This part is analysed? still issues there?

Click2Cloud-Gamma replied:
No we have tested with latest Elba release too, it works fine. Thanks

Click2Cloud-Gamma commented:
> @Click2Cloud-Gamma
> Why there is no description to the PR?
> How about the previous PR #51 comments are closed? no comment updates

@skdwriting  Yes, I have added all the requested changes in this design-spec document. Thanks

skdwriting commented:
The previous review comments are closed. Unable to see it from here as the earlier PR is closed.. Could you please check with reviewers?

Click2Cloud-Gamma commented:
> The previous review comments are closed. Unable to see it from here as the earlier PR is closed.. Could you please check with reviewers?

FIxed all the requested changes and moved files to `specs` folder as requested. Please review, Thanks
